doc/subscribe.rst

```diff
@@ -7,11 +7,12 @@ Subscribe
 Dipy_ is a part of the Nipy_ community and we are happy to share the same
 e-mail list. This makes sense as we can share ideas with a broader
 community and boost collaboration across neuroimagers. If you
-want to post to the list you will need first to subscribe to `nipy
+want to post to the list you will need first to subscribe to the `nipy
 mailing list`_. We suggest to begin the subject of your e-mail with
 ``[Dipy]``. This will help us respond faster to your questions.
 
-Best wishes,
-Dipy developers
+Additional help can be found on the Neurostars_ website. You can 
+sometimes catch us on the `dipy gitter`_ channel.
```

Comment:
OK - sent.

On Sat, Nov 21, 2015 at 3:05 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In doc/subscribe.rst
> <https://github.com/nipy/dipy/pull/773#discussion_r45552197>:
>
> >  mailing list`_. We suggest to begin the subject of your e-mail with
> >  ``[Dipy]``. This will help us respond faster to your questions.
> >
> > -Best wishes,
> > -Dipy developers
> > +Additional help can be found on the Neurostars_ website. You can
> > +sometimes catch us on the `dipy gitter`_ channel.
>
> Sure I understand your point. I suggest that you send what you wrote above
> in an e-mail so that as many people know about this experiment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/773/files#r45552197>.
>